doc/test-suite.py

```diff
@@ -77,30 +75,39 @@
                 continue
 
             # reset the options
-            options = {**defaultOptions, **testOptions}
+            options = testOptions.copy()
 
             # filenames (input MEI/XML and output SVG)
             inputFile = os.path.join(path1, item1, item2)
-            print(inputFile)
+            options.update({"xmlIdSeed": int(hashlib.sha256(
+                inputFile.encode("utf-8")).hexdigest(), 16) % 10**9})
+            print(f'Rendering {item2}')
             name, ext = os.path.splitext(item2)
             svgFile = os.path.join(path2, item1, name + '.svg')
             pngFile = os.path.join(path2, item1, name + '.png')
+            timeMapFile = os.path.join(path2, item1, name + '.json')
 
             # parse the MEI file
             if ext == '.mei':
                 tree = ET.parse(inputFile)
                 root = tree.getroot()
                 # try to get the extMeta tag and load the options if existing
-                meta = root.findtext('.//mei:meiHead/mei:extMeta', namespaces=ns)
+                meta = root.findtext(
+                    './/mei:meiHead/mei:extMeta', namespaces=ns)
                 if meta is not None and meta != '':
                     print(meta)
                     metaOptions = json.loads(meta)
-                    options = {**options, **metaOptions}
+                    options |= metaOptions
 
             tk.setOptions(json.dumps(options))
             tk.loadFile(inputFile)
+            # render to SVG
             svgString = tk.renderToSVG(1)
             svgString = svgString.replace(
                 "overflow=\"inherit\"", "overflow=\"visible\"")
             ET.ElementTree(ET.fromstring(svgString)).write(svgFile)
             cairosvg.svg2png(bytestring=svgString, scale=2, write_to=pngFile)
+            tk.resetOptions()
```

Comment:
Why resetting the options before rendering the timemap?

Reply:
Without reason.